addon/content/new-tab-variation.js

```diff
@@ -111,6 +113,7 @@ class TrackingProtectionStudy {
       }
     }
     if (timeMinutes > 0) {
+      // eslint-disable-next-line no-nested-ternary
```

Comment:
Disabled the ternary warning here since it didn't feel grand enough that we'd need to refactor out into a separate variable to hold a comma or the word "and".
If we have to support l10n eventually, we'd probably have to refactor this bit anyways to properly display times in a locale aware way.